backend/tests/websockets/user_confirmer_test.py

```diff
@@ -1,73 +1,75 @@
-import logging
-from unittest.mock import Mock, patch
+# import logging
```

Comment:
If these tests aren't needed any more then they should be deleted and not just commented out.

Reply:
Tests deleted